packages/twenty-website/src/app/api/contributors/contributorStats/[slug]/route.tsx

```diff
@@ -0,0 +1,23 @@
+import { getContributorActivity } from '@/app/contributors/utils/get-contributor-activity';
+
+export const dynamic = 'force-dynamic';
+
+export async function GET(request: Request) {
```

Comment:
The issue is that this isn't going to be accurate at the moment it's called with our cron? or will it?

Reply:
Indeed, I had already thought this through. In my opinion it wouldn’t be an issue because the interval between the creation of a PR, its review, and subsequent merging is likely to be longer than the 5-minute interval of the scheduled cron job.

Reply:
It would if we were showing "Total PRs" but we show "Merged PRs" on the card.
Maybe we should show Total PRs instead? It emphasizes quality even less but I guess that's the best quickfix I can think of :(.

Reply:
You're right, it's problematic ! Okay I can change it to "TotalPRs" for now until a better solution is found

Reply:
The rank would also have to be changed as it is based on mergedPRs :(

Reply:
arf then maybe expose an endpoint which you call before pre-fetching the image to launch a re-import for that user? and we keep range/total merged PRs

Reply:
Any chance we can do a github:sync ? How often do we merge PRs from contributors (for rate limits)?

Reply:
Yes sharing the code from github:sync works, then we'll re-expose it as an endpoint! We can keep both
I was thinking of adding an additional param to limit the sync (had we kept the searchIssue partial sync mechanism it would have been one more search parameter in the partial sync, probably also possible in the current script - but ok to not do it if it happens to be complex)

Reply:
Okay perfect !